tests/README.md

```diff
@@ -43,6 +43,8 @@ Any time changes made to the operator are being checked into the main branch, sa
   - (if running sanity, powerflex, or modules suites) `test-vxflexos`
   - (if running sanity, powerscale, or modules suites) `isilon`
   - (if running unity suite) `unity`
+  - (if running powermax suite) powermax
```

Comment:
can we put the namespace in tildes to match above, e.g. `powerstore`

Reply:
added!